tests/specs/run/package_json/invalid_value/error.ts.out

```diff
@@ -1,5 +1,2 @@
-error: Invalid version requirement
-    0: Unexpected character.
-         invalid stuff that won't parse
-         ~
+error: Could not resolve "@denotest/cjs-default-export", but found it in a package.json. Deno expects the node_modules/ directory to be up to date. Did you forget to run `deno install`?
```

Comment:
we no longer receive a helpful error message like we previously did. I'll probably fix the error message.

cc @dsherret

Reply:
I think we should be at least displaying a warning for `deno install` here. I wonder why we don't.